package.json

```diff
@@ -41,7 +41,8 @@
     "tmp": "^0.0.25",
     "z-schema": "^3.12.0",
     "chai": "^3.0.0",
-    "mocha": "^2.2.5"
+    "mocha": "^2.2.5",
+    "request": "^2.58.0"
```

Comment:
Why request is a dependency? To run the test generator code you don't need request.